### PR TITLE
Column description for the Shipment page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds missing columns description for the Shipment page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/sales/shipments.html

### CODE REVIEW

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

File: src/sales/shipments.md [hunk 1]
```diff
@@ -8,3 +8,28 @@ Magento supports partial and complete order shipment, with additional options av
 
 ![]({% link images/images/sales-shipments.png %}){: .zoom}
 _Shipments grid_
+
+## Column descriptions
+
+|Control|Description|
+|--- |--- |
+|Select|Select the checkbox to select the quote(s) to be subject to an action, or use the selection control in the column header. Options: Select All / Deselect All|
```
Comment (leslietilling) [inline src/sales/shipments.md:16]:
```suggestion
|Select|Select the checkbox for each quote to be subject to an action, or use the selection control in the column header. Options: Select All / Deselect All|
```

File: src/sales/shipments.md [hunk 2]
```diff
@@ -8,3 +8,28 @@ Magento supports partial and complete order shipment, with additional options av
...
+|Shipment|A unique, sequential number that is assigned when a new shipment is saved for the first time|
+|Ship Date|Shipping date|
+|Order|Unique number of the order|
+|Order Date|The date and time the order was placed|
+|Ship-to Name|The name of the person to whom the order is shipped|
+|Total Quantity|Total quantity of items to ship|
+|Action|View opens the shipment in edit mode|
+
+### Additional Columns
```
Comment (leslietilling) [inline src/sales/shipments.md:25]:
```suggestion
### Additional columns
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**